src/components/index.js

```diff
@@ -455,7 +455,7 @@ export default function SlateTranscriptEditor(props) {
   };
   return (
     <Container fluid style={{ backgroundColor: '#eee', height: '100vh', paddingTop: '1em' }}>
-      <style scoped>
+      <style scope>
```

Comment:
```suggestion
      <style scoped>
```
I think it's with a `d`?

http://html5doctor.com/the-scoped-attribute/

Reply:
and line 466 was the one with the typo

Reply:
Oops! Apologies. Just a moment.

Reply:
no probl at all, and no rush 😄

Reply:
Done :)